pkg/policy/rules.go

```diff
@@ -380,15 +380,3 @@ func (rules ruleSlice) updateEndpointsCaches(ep Endpoint, epIDSet *IDSet) bool {
 
 	return false
 }
-
-func (rules ruleSlice) refreshRulesCache(ep Endpoint) {
```

Comment:
This one's my fault, missed that it became unused. See 2d9fdde139f9 ("daemon: Don't populate rule selector cache on restore")